lfucg-exactions/client/js/components/PlatSummary.js

```diff
@@ -170,20 +170,20 @@ class PlatSummary extends React.Component {
                                         </div>
                                     </div>
                                     <div className="col-xs-12">
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Plat Name: {plats.name}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Plat Type: {plats.plat_type_display}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Subdivision: {plats.subdivision ? plats.subdivision.name : null}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Approval: {plats.is_approved ? 'Approved' : 'Not Approved'}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Expansion Area: {plats.expansion_area}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Unit: {plats.unit}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Section: {plats.section}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Block: {plats.block}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Cabinet: {plats.cabinet}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Slide: {plats.slide}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Buildable Lots: {plats.buildable_lots}</p>
-                                        <p className="col-md-3 col-sm-4 col-xs-6">Non-Buildable Lots: {plats.non_buildable_lots}</p>
-                                        <p className="col-sm-4 col-xs-6">Current Sewer Due: {plats.plat_exactions && plats.plat_exactions.plat_sewer_due}</p>
-                                        <p className="col-sm-4 col-xs-6">Current Non-Sewer Due: {plats.plat_exactions && plats.plat_exactions.plat_non_sewer_due}</p>
+                                        <h3 className="col-xs-6">Plat: {plats.cabinet}-{plats.slide}</h3>
+                                        <h3 className="col-xs-6">{plats.is_approved ? 'Approved' : 'Not Approved'}</h3>
+                                        <p className="col-xs-6">Name: {plats.name ? plats.name : null}</p>
+                                        <p className="col-xs-6">Subdivision: {plats.subdivision ? plats.subdivision.name : null}</p>
```

Comment:
Can use && here instead of allowing for null.  Either extend the test around the label or return 'None' as better indication.